### PR TITLE
New 'Governance' page

### PR DESCRIPTION
**Pull request by BreadJS**
## Abstract
I've changed the design of the 'Governance' page to make it more responsive and have a cleaner UI.

## What does this PR address?
MPW was not really responsive for other devices and lacks a good design (IMO). My intent is to redesign MPW for a better look and also make it as responsive as possible for every device.

## What features or improvements were added?
1. New `Governance` page
    - Added `Monthly Budget`
    - Added `Budget Allocated`
    - Added `Next Treasury Payout`
    - New table design
    - Responsive for mobile

## How does this benefit users?
A better design will lure more users in using MPW. This also applies for making the site more responsive.

## PIVX Address
`DDS5v6mPqUN3rdzkp3268LV7TKjHCZqy8c`

### CODE REVIEW

**Review by Liquid369 — APPROVED** [review]
tACK f91f1ab0f30ec78237b1c39325db0a9eee22a1bf
Looks great on Mac this time!! Safari no issues.
Codewise ACK
Compared data between pivx.org and MPW and everything is proper 
I tested proposal creation on test net a-ok
Do not have a node to test voting success at the moment but GG

**Review by SKDTXL — APPROVED** [review]
tACK f91f1ab0f30ec78237b1c39325db0a9eee22a1bf
Looking pretty much awesome.
UI is outstanding with tons of new data and functionality.
Tested on both desktop and mobile environment, both tests were pretty smooth and 100% mobile friendly.
I also compared data between the (PIVX Treasury Dashboard), everything looks accurate to me, and you'll find all Governance data you'd ever need.
I couldn't test voting success because this functionality is not available in TestNet. But I pretty much think it's good to go.

File: scripts/flipdown.js
Comment (Duddino) [inline scripts/flipdown.js:1]: This should be included as an npm dependency instead

File: scripts/global.js [hunk 1]
```diff
@@ -431,6 +452,7 @@ function subscribeToNetworkEvents() {
 // WALLET STATE DATA
 export const mempool = new Mempool();
 let exportHidden = false;
+let governanceFlipdown = null;
```
Comment (Duddino) [inline scripts/global.js:455]: Jsdoc typings would be useful here

File: scripts/global.js [hunk 2]
```diff
@@ -506,28 +559,12 @@ export function updateTicker() {
...
             ((fCold ? getStakingBalance() : getBalance()) / COIN) * nPrice;
 
-        // Handle certain edge-cases; like satoshis having decimals.
-        switch (strCurrency) {
-            case 'sats':
-                nValue = Math.round(nValue);
-                cLocale.maximumFractionDigits = 0;
-                cLocale.minimumFractionDigits = 0;
-        }
+        // Handle certain locale display edge-cases; like satoshis having decimals.
```
Comment (Duddino) [inline scripts/global.js:566]:
```suggestion
```
This seems like a leftover comment

File: assets/style/style.css [hunk 3]
```diff
@@ -2884,3 +2884,423 @@ select.form-control option {
 #foot {
     z-index: 200;
 }
+
+
+
+/* Flipdown styling (flipdown.js library) */
```
Comment (Duddino) [inline assets/style/style.css:2890]: This should be included as an npm  dependencyinstead

**Review by Duddino — DISMISSED** [review]
I like the overall appearance, godo job. I have a couple of issues though.
I noticed `flipdown.js` was included directly instead of using the [npm package](https://www.npmjs.com/package/flipdown), is there a specific reason for this? 
Also, you can't finalize proposals in this PR (The button is gone)
The timer doesn't work on testnet and it stays the same when you switch network

File: assets/style/style.css (hunk 3 above)
Comment (BreadJS) [reply assets/style/style.css:2890]: I'm aware. We tried it. But it has no ES6 support so it does not work.

**Comment by JSKitty** [comment]
There's minor CSS conflicts to resolve here, along with JSDoc typing. (for `governanceFlipdown`)

File: scripts/global.js (hunk 1 above)
Comment (BreadJS) [reply scripts/global.js]: Done that! :)

**Comment by BreadJS** [comment]
Requests resolved:
- Added `governanceFlipdown` JSDoc typing
- Added finalise button on proposals
- Flipdown counter updates on network switching

**Review by Duddino — APPROVED** [review]
tACK. The only issue remaining is that of including `flipdown.js` in MPW which I'm not a fan of. Maybe we can fork the library and add module support ourselves, but I would leave it for another PR

**Review by JSKitty — APPROVED** [review]
**tACK** bf8d44e9bad86ef19317423e5196e38a349bcbe9

:tada: Beautiful UI, extremely nice features and design - I'm also happy that we finally clarified the finalisation process, as it's always been pretty unclear and awkward with UX.

I have tested this fairly thoroughly on Testnet, with no issues or nits.

In the future, we should redesign the Proposal Creation menu (as the current one is basically a glorified Prompt popup), but that is out of scope for this PR.

**Review by Liquid369 — APPROVED** [review]
tACK bf8d44e9bad86ef19317423e5196e38a349bcbe9

These improvements are great, the finalization works well, and is noticeable when setting up a new proposal. Vast improvement gg

**Pull request merged**